feeder/ethereum/src/Feeder.ts

```diff
@@ -0,0 +1,76 @@
+import RootWitnessClient from "./RootWitnessClient";
+import { hashString, Log, UpdateData } from "./utils";
+import { BigNumber, ethers } from "ethers";
+import LogClient from "./LogClient";
+
+// Feeder encapsulates functionality related to feeding
+// Ethereum-based RootWitness logs updates.
+export class Feeder {
```

Comment:
How about `CTFeeder`, or otherwise specifying that this is CT somewhere in this file path?
Currently the path to this class is "feeder/ethereum/src/Feeder.ts/Feeder", which looks quite generic.

Reply:
Done